packages/session-replay-browser/src/helpers.ts

```diff
@@ -31,3 +31,23 @@ export const getCurrentUrl = () => {
   const globalScope = getGlobalScope();
   return globalScope?.location ? globalScope.location.href : '';
 };
+
+export const generateSessionReplayId = (sessionId: number, deviceId: string): string => {
+  return `${deviceId}/${sessionId}`;
+};
+
+export const parseSessionReplayId = (
+  sessionReplayId: string | undefined,
+): { deviceId?: string; sessionId?: string } => {
+  if (!sessionReplayId) {
+    return {};
+  }
+
+  const parts = sessionReplayId.split('/');
```

Comment:
this is a bit brittle, can break if customer passes in the wrong sessionReplayId that's not made of device/session id. I wonder if we should drop the option to directly pass in a SessionReplayId before we can better support it.